packages/prosess-beregningsgrunnlag/src/components/BeregningFP.jsx

```diff
@@ -241,6 +241,7 @@ const formaterAksjonspunkter = aksjonspunkter =>
         {
           '@type': kode,
           ...aksjonspunkt,
+          periode: perioder.find(p => p.periode.fom === aksjonspunkt.skjæringstidspunkt).periode,
```

Comment:
Er det mulig at perioder er en tom array eller at vi ikke finner en match her? mulig TypeError her også isåfall.

Reply:
Då er det i så fall noko som har gått galt backend, så eg tenkjer det er greit å feile.